doc/modules/linear_model.rst

```diff
@@ -126,6 +126,23 @@ its ``coef_`` member::
     >>> reg.intercept_
     0.13636...
 
+Note that the class :class:`Ridge` allows for the user to specify that the 
+solver be automatically chosen by setting `solver="auto"`. When this option 
+is specified, :class:`Ridge` will choose between the `"lbfgs"`, `"cholesky"`, 
+and `"sparse_cg"` solvers. :class:`Ridge` will begin checking the conditions
+shown in the following table from top to bottom. If the condition is true,
+the corresponding solver is chosen.
+
++-------------+----------------------------------------------------+
+| **Solver**  | **Condition**                                      |
++-------------+----------------------------------------------------+
+| 'lbfgs'     | The ``positive=True`` option is specified.         |
++-------------+----------------------------------------------------+
+| 'cholesky'  | The input array X is not sparse.                   |
++-------------+----------------------------------------------------+
+| 'sparse_cg' | All of the above conditions are not fulfilled.     |
```

Comment:
The wording here feels a bit odd to me. I would prefer something like 
```suggestion
| 'sparse_cg' | None of the above conditions are fulfilled.        |
```

Reply:
Good point! That's definitely more direct. Thank you!